pkg/kubectl/cmd/drain.go

```diff
@@ -149,6 +152,9 @@ var (
 		ReplicaSet, DaemonSet or Job, then drain will not delete any pods unless you
```

Comment:
Should we mention StatefulSet here, in the example, and in the flag description?